src/IECoreHoudini/bindings/FnParameterisedHolderBinding.cpp

```diff
@@ -36,13 +36,154 @@
 
 #include "IECoreHoudini/bindings/FnParameterisedHolderBinding.h"
 
-#include "IECoreHoudini/FnParameterisedHolder.h"
+#include "IECoreHoudini/NodeHandle.h"
+#include "IECoreHoudini/ParameterisedHolderInterface.h"
 
 #include "OP/OP_Node.h"
 
 using namespace boost::python;
 using namespace IECoreHoudini;
 
+namespace
+{
+
+class FnParameterisedHolder
```

Comment:
Is there no possibility of wanting to use an FnParameterisedHolder from C++?

Reply:
The comment on the original header was `/// \todo: This class is a bit funny. Can this all be moved to the bindings like in IECoreMaya?`.

I looked at the Maya equivalents and the only `IECoreMaya` namespaced functions are `parameterisedHolderAssignModificationState` and `bindFnParameterisedHolder`. There are a bunch of functions that aren't anonymous namspaced, but I'd assumed that's because they were written before we adopted that convention.

Also, this is just wrapping functions from `IECoreHoudini::ParameterisedHolder` with an "alive" flag, so a C++ user can use the node directly without needing the `Fn` and ask if its alive using the HDK (c++ api).

Reply:
> this is just wrapping functions from IECoreHoudini::ParameterisedHolder with an "alive" flag, so a C++ user can use the node directly without needing the Fn and ask if its alive using the HDK (c++ api).

Ah - got it. I'd looked at SOP_ParameterisedHolder and noted it didn't have `setParameterised()` but then hadn't taken the time to realise it got it from the base class. Thanks for clarifying.